js/src/components/paid-ads/asset-group/assets-loader.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import { useState, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
+import AppButton from '.~/components/app-button';
+import SelectControl from '.~/wcdl/select-control';
+import { API_NAMESPACE } from '.~/data/constants';
+import './assets-loader.scss';
+
+/**
+ * @typedef {import('.~/data/types.js').AssetGroup} AssetGroup
+ */
+
+function allowSuggestedPages( search ) {
+	// Querying with empty `search` means getting the suggestion pages by default.
+	// Make it result in `new RegExp('.', 'i')` to avoid the default results being filtered out.
+	if ( search === '' ) {
+		return '.';
+	}
+	return search;
```

Comment:
💅  Maybe we can shorten this as `return search || '.' `

Reply:
The original implementation was changed in https://github.com/woocommerce/google-listings-and-ads/pull/1805/commits/f084e1a13d30b8243e8f19de624fca7b54993540 while addressing another review comment https://github.com/woocommerce/google-listings-and-ads/pull/1805#issuecomment-1343826955